libs/app-example-api-services/src/lib/appointment-api.service.ts

```diff
@@ -42,38 +46,25 @@ export class AppointmentApiService {
                 params = params.set('orderBy', filterOptions.orderBy)
             }
         }
-        return this.http.get<ApiResponse<Appointment[]>>(
-            `${this.apiUrl}/appointments`,
-            { params },
-        )
+        return this.find(params)
     }
 
     getAppointment(id: string): Observable<ApiResponse<Appointment>> {
-        return this.http.get<ApiResponse<Appointment>>(
-            `${this.apiUrl}/appointments/${id}`,
-        )
+        return this.findById(id)
```

Comment:
Instead of wrapping the `findById` method by `getAppointment` method, rather use the `findById` method directly. So, you can delete the `getAppointment` method and use findById. Do the same for rest of the methods as well.